apisix/plugins/azure-functions.lua

```diff
@@ -14,30 +14,15 @@
 -- See the License for the specific language governing permissions and
 -- limitations under the License.
 
-local core = require("apisix.core")
-local http = require("resty.http")
 local plugin = require("apisix.plugin")
-local ngx  = ngx
-local plugin_name = "azure-functions"
+local plugin_name, plugin_version, priority = "azure-functions", 0.1, -1900
 
-local schema = {
+local azure_authz_schema = {
```

Comment:
we are developing a new feature "aws lambda serverless", so we should not update `azure-functions.lua`, it is a different thing.

one for one thing.

Reply:
I agree. This plugin depends on the refactoring proposed in #5616. For sake of fast development, I developed the feature on top of the branch ([ref](https://github.com/apache/apisix/pull/5594#issuecomment-979496919)). As the changes in 5616 has been merged into master, I have rebased it. Thanks : )